Sources/ConsoleKit/Activity/ActivityBar.swift

```diff
@@ -35,12 +37,12 @@ extension ActivityBar {
 }
 
 /// Defines the width of all `ActivityBar`s in characters.
-private var _width: Int = 25
+private let _width: ManagedAtomic<Int> = ManagedAtomic(25)
 
 extension ActivityBar {
     /// Defines the width of all `ActivityBar`s in characters.
     public static var width: Int {
-        get { return _width }
-        set { _width = newValue}
+		get { return _width.load(ordering: .relaxed) }
+		set { _width.store(newValue, ordering: .relaxed)}
```

Comment:
In addition to my remarks above, both `LoadingBar.swift` (line 45) and `ProgressBar.swift` (lines 54 and 55) incorrectly refer to the global property instead of the `width` parameter, this should be fixed.